team-01/src/main/java/ubc/cosc322/MonteCarloPlayer.java

```diff
@@ -33,40 +33,42 @@
  * 2. Fullstack Academy, https://www.youtube.com/watch?v=Fbs4lnGLS8M
  */
 public class MonteCarloPlayer extends LocalPlayer {
-	private final long MAX_RUNTIME = 10000;
+	private final long MAX_RUNTIME = 25000;
 
 	// The constant used for UCB function. Same one chosen in the John Levine video.
 	private final double EXPLORATION_FACTOR = Math.sqrt(2);
 
 	private TreeNode root;
 
 	public MonteCarloPlayer() {
-		super("jaden_mc", "montecarlo");
+		super("group1-mc", "montecarlo");
 	}
 
 	@Override
 	protected void onMoveReceived() {
 		AmazonsLocalBoard rootState = board.copy();
-		rootState.localPlayer = board.getOpponent();
```

Comment:
- Caused our agent to move the opposing teams player, handled the initial reason for this in getBestMove